specification/appendix_d.adoc

```diff
@@ -0,0 +1,88 @@
+during the boot of the platform, resulting in the following advantages: (1) simplified formal reasoning about the 
+ownership of memory, (2) attestation that covers static system configuration (e.g., values of PMP registers), 
+(3) reduced attack surface between OS/VMM and TSM (narrower ABI). A possible negative consequence of 
+static partitioning is underutilization of resources.
+
+=== TVM creation
+To reduce the complexity of the TSM implementation, the TSM creates a TVM as a result of a single operation triggered with 
+the `sbi_covg_promote_to_tvm()` call. Specifically, OS/VMM initializes and starts a regular VM, which early during the 
+boot process requests to be promoted to TVM. This call is reflected to the TSM which copies TVM data, the page table 
+configuration, and the vCPU state into confidential memory. 
+
+=== Local attestation
+Embedded systems might operate without access to a network, which prevents use of remote attestation. For this 
+reason, this deployment model also supports local attestation, in which the TSM attests to the integrity of the TVM image 
+during its creation and allows its creation only when it contains a specfic local attestation payload (LAP). This 
+payload carries a cryptographic proof issued with the expected attestation key specific to the TSM integrity 
+and platform configuration. The call to promote to a TVM requires two parameters: (1) a pointer to the flatten device tree (FDT) 
+and (2) a pointer to LAP. If the pointer to LAP is zero, then remote attestation is used, otherwise local attestation is used. 
+Local attestation is strongest when it is hardware enforced.  
+
+Attestation for embedded systems utilizes one or both of the following properties. 
+
+. The embedded platform must be able to verify that the VM/TVM is authorized to run on the platform. 
+. The TVM must be able to verify that the configuration of the hardware platform is acceptable/correct.
+
+Separate mechanisms may be used to achieve these goals.
+
+====VM (TVM) Authorization
+The TSM must have a list of public keys of those authorized to sign VMs (TVMs) for execution on the platform. The 
+local attestation payload associated with the TVM will be
+signed with a private key. When the VM request to be promoted to a TVM, TSM checks the signature on the LAP. 
+If the signature is not valid, the TSM will not convert the VM and will terminate execution of the
+VM. The method for provisioning these public keys into the TSM is outside the scope of this specification.
+
+====Verifying Platform configuration
+When the creator of the (authorized) TVM does not want it to execute on improperly configured or unauthorized hardware,
+there should be a mechanism supported by hardware (and firmware) for verification. 
+Assuming such hardware exists, for example a trusted platform module (TPM), the VM can be created with an encrypted disk. 
```

Comment:
perhaps better to use a generic term like HW root-of-trust of measurement, storage